core/src/commonMain/kotlin/dev/fritz2/core/lens.kt

```diff
@@ -6,6 +6,15 @@ package dev.fritz2.core
 @Target(AnnotationTarget.CLASS)
 annotation class Lenses
 
+/**
+ * Used by the fritz2 gradle-plugin to identify properties in sealed classes or interfaces, that should get ignored
+ * by the lens generation.
+ *
+ * Typical use case are const properties, that are override inside the data class body and not the ctor.
```

Comment:
Instead of `override`, `overridden` would be the grammatically correct choice here